### ISSUE TITLE
flow-cli 0.40.0 `invalid default access check mode` issue

### ISSUE THREAD

**Issue by chenrui333** [onflow/flow-cli#663]
### Problem

Have seen some runtime issue after building out `flow-cli` 0.40.0 release.

### Steps to Reproduce

- 1. grab the code archive, `https://github.com/onflow/flow-cli/archive/refs/tags/v0.40.0.tar.gz`
- 2. go build
- 3. run `./flow cadence hello.cdc`

Then you would see

```
$ ./flow cadence hello.cdc
error: invalid default access check mode
--> hello.cdc
```

### Acceptance Criteria

Passed `./flow cadence hello.cdc` test

### Context

relates to https://github.com/Homebrew/homebrew-core/pull/110886
This is similar to what I saw in cadence, https://github.com/onflow/cadence/issues/1912

**Comment by sideninja** [comment]
Thank you for reporting, looking into this now.

**Comment by bluesign** [comment]
Cadence 0.26.1 fixed this. https://github.com/onflow/cadence/releases/tag/v0.26.1

**Comment by turbolent** [comment]
This should be fixed by the latest release

**Comment by chenrui333** [comment]
Yep, confirmed on my side, https://github.com/Homebrew/homebrew-core/pull/111813
